### DISCUSSION TITLE
请问我可以使用vite进行开发吗

### DISCUSSION THREAD

**Discussion by nyaruhodoo** [NervJS/taro#11037] [Q & A]
请问我可以使用vite进行开发吗

**Comment by Chen-jj** [comment]
还没支持的

**Reply by youmengme** [reply]
最近有新的进度吗了 大佬

**Reply by Chen-jj** [reply]
Q2 开始跟进的

**Comment by Mrcxt** [comment]
还有计划支持吗，23年了。。。

**Reply by AdvancedCat** [reply]
应该是计划支持的。
不过据 @Chen-jj  自述某天 coding 风驰电掣，把电脑干崩了，taro-vite 毁于一旦，无奈从头再来。
要养成每天提交代码的好习惯。

**Comment by youmengme** [comment]
期待ing...

**Comment by ruidoc** [comment]
请问有仓库吗，可以参与呀

**Comment by Chen-jj** [comment]
关于实现细节的讨论在：[NervJS/taro-rfcs#18](https://github.com/NervJS/taro-rfcs/pull/18)

具体实现进度：https://tarojs.notion.site/Vite-eb5c8c48a57e48778449901ff90041d8

分支在 https://github.com/NervJS/taro/tree/feat/3.7-vite

最近事情较多，迭代速度较为缓慢

**Comment by ingf** [comment]
请问预期发布时间会在几月份呀

**Comment by Miofly** [comment]
期待

**Comment by xyy94813** [comment]
mark vite branch:

[feat/vite-h5](https://github.com/NervJS/taro/tree/feat/vite-h5)
[feat/vite-vue2](https://github.com/NervJS/taro/tree/feat/vite-vue2)
[feat/vite-h5-react](https://github.com/NervJS/taro/tree/feat/vite-h5-react)
[feat/3.7-vite](https://github.com/NervJS/taro/tree/feat/3.7-vite)
